.env_example

```diff
@@ -1,4 +1,5 @@
 SPOTIFY_CLIENT_ID = SPOTIFY_CLIENT_ID
 SPOTIFY_SECRET_ID = SPOTIFY_SECRET_ID
 ADMIN_EMAIL=ADMIN_EMAIL
-SENTRY_DSN=SENTRY_DSN
\ No newline at end of file
+SENTRY_DSN=SENTRY_DSN
+REDIS_URL=redis://redis:6379/1
```

Comment:
why do we need it here?

Reply:
For production it is necessary to set to avoid default value, at least when someone see they will know that they have to run a redis server also.